### ISSUE TITLE
Have an option to stop containers without having a prompt 

### ISSUE THREAD

**Issue by peauc** [jesseduffield/lazydocker#558]
When working on some environments I have to start and stop containers rapidly. I believe adding a config option to disable to confirm prompt on stop would be useful.

**Comment by blwsh** [comment]
Might be a useful config option but for your use case of stopping and starting (assuming immediate restart) just use the restart shortcut (r) which does not prompt you when stopping the container.

**Comment by peauc** [comment]
@blwsh I don't want to restart a started container but start a stoped container

**Comment by blwsh** [comment]
Hitting restart on a stopped container will start the container.

**Comment by peauc** [comment]
Yes that's the issue. I dont want to `docker restart` but `docker start`
